### PR TITLE
EPAS: components prototype

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Related to this feedback in Slack: https://edb.slack.com/archives/CU5QEU5L7/p1673794294714659
>What do people think about potentially having a "supporting components" tab on https://www.enterprisedb.com/docs/epas/latest/
Having searched the docs, the information is available here https://www.enterprisedb.com/docs/epas/latest/installing/linux_install_details/rpm_packages/#edb-postgres-advanced-server-rpm-packages, but it's not as easily discoverable
I personally think it's useful how on https://www.enterprisedb.com/docs/pgbouncer/latest/supported_platforms/ &
https://www.enterprisedb.com/docs/odbc_connector/latest/02_requirements_overview/ it's easily discoverable that it's used with EPAS only
We had a ticket recently where the customer was using EPAS with open source pgbouncer, and experienced errors as a result

Added a top-level topic

### CODE REVIEW

File: product_docs/docs/epas/14/components.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: EDB Components
+---
+
+We package and support a number of Postgres components to use with EDB Postgres Advanced Server. The EDB components support the advanced features of EDB Postgres Advanced Server (needs PM review). EDB Postgres Advanced Server also works with the community versions of these components.
+
+The EDB components include:
+
+- EDB PostGIS (links to the docs)
+- EDB Connectors
```
Comment (arup-roy2023) [inline product_docs/docs/epas/14/components.mdx:10]:
```suggestion
- EDB Connectors_Test
```

File: product_docs/docs/epas/14/data_sources.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Connecting to external data sources
+---
+
+We package and support a number of components to use to connect your EDB Postgres Advanced Server database server to external data sources. These EDB components are based on PostgreSQL components and typically support additional EDB Postgres Advanced Server capabilities. 
+
+The EDB components include:
+
+  - [EDB Hadoop Foreign Data Wrapper](/jdbc_connector/latest/installing/) &mdash; allows access to data that resides on a Hadoop file system from a Postgres database server
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/data_sources.mdx:9]:
```suggestion
  - [EDB Hadoop Foreign Data Wrapper](/jdbc_connector/latest/installing/) &mdash;  accesses data that resides on a Hadoop file system from a Postgres database server
```

File: product_docs/docs/epas/14/index.mdx [hunk 3]
```diff
@@ -34,10 +35,15 @@ navigation:
...
+  - epas_qs_linux_8
+  - epas_qs_windows
 legacyRedirectsGenerated:
   # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
   - "/edb-docs/p/edb-postgres-advanced-server/9.6"
   - "/edb-docs/p/edb-postgres-advanced-server/9.5"
 ---
 
-  With EDB Postgres Advanced Server, EnterpriseDB continues to lead as the only worldwide company to deliver innovative and low-cost, open-source-derived database solutions with commercial quality, ease of use, compatibility, scalability, and performance for small or large-scale enterprises.
+  With EDB Postgres Advanced Server, we continues to lead as the only worldwide company to deliver innovative and low-cost, open-source-derived database solutions with commercial quality, ease of use, compatibility, scalability, and performance for small or large-scale enterprises.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/index.mdx:48]:
```suggestion
  With EDB Postgres Advanced Server, we continue to lead as the only worldwide company to deliver innovative and low-cost, open-source-derived database solutions with commercial quality, ease of use, compatibility, scalability, and performance for small or large-scale enterprises.
```

File: product_docs/docs/epas/14/data_sources.mdx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+title: Connecting to external data sources
+---
+
+We package and support a number of components to use to connect your EDB Postgres Advanced Server database server to external data sources. These EDB components are based on PostgreSQL components and typically support additional EDB Postgres Advanced Server capabilities.
```
[the remark this answers is not in the document]
Comment (nidhibhammar) [reply product_docs/docs/epas/14/data_sources.mdx:5]: +1

File: product_docs/docs/epas/14/managing_conections.mdx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+title: Managing your database connections
+---
+
+We package and support a number of components to manage your connections to your EDB Postgres Advanced Server database server. These EDB components are based on PostgreSQL components and typically support additional EDB Postgres Advanced Server capabilities.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/managing_conections.mdx:5]: Same comment as mentioned above by Josh

**Review by ccestes — APPROVED** [review]

**Pull request merged**